packages/gatsby-image/withIEPolyfill/index.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
```

Comment:
you should add this file to gitignore as we don't commit compiled sources.

Reply:
Will do!

I notice that the compiled version of the main image element (`packages/gatsby-image/index.js`) doesn't appear to be included in the `.gitignore` file... Should I add it as well, or is this wrapper component handled differently?

Reply:
sorry for giving you half baked answers.

`/*.js` is only handling direct children and because `withIEPolyfill/index.js` is inside a folder it's not being handled. You can change it into
```
/**/*.js
!/src/**/*.js
```
to also check subdirectories but keep the src folder in version control but for your usecase it's porbably enough to just disable withIEPolyfill like `/withIEPolyfill/*.js`

Reply:
I see! Thanks so much for that walkthrough.